testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -1,12 +1,15 @@
-import MDAnalysis as mda
 import numpy as np
 from six.moves import zip
-from MDAnalysis.coordinates.base import Timestep
-
-from numpy.testing import (assert_equal, assert_raises, assert_almost_equal,
-                           assert_array_almost_equal, raises)
+from nose.plugins.attrib import attr
```

Comment:
In the main package we're doing:
 - global/general
 - local

So it would then make sense to maybe do
 - global
 - MDAnalysis
 - MDAnalysisTests

I think that's what I tried to do